operations/mimir/ruler.libsonnet

```diff
@@ -30,6 +30,8 @@
       'ruler.ring.consul.hostname': 'consul.%(namespace)s.svc.%(cluster_domain)s:8500' % $._config,
 
       'server.http-listen-port': $._config.server_http_port,
+
+      'distributor.remote-timeout': '10s',
```

Comment:
[nit] I suggest to add a comment to explain "why" (basically what you say in the PR description: "... to accommodate writing large rule results to the ingester").